generated/nidaqmx/system/storage/_alternate_task_constructor.py

```diff
@@ -17,14 +17,17 @@ class _TaskAlternateConstructor(Task):
     derived class to 'overload' the constructor.
     """
 
-    def __init__(self, task_handle):
+    def __init__(self, task_handle,*, grpc_options=None, interpreter=None):
```

Comment:
This class is used to construct with an existing task handle, so I think you should simplify it to:
```
def __init__(self, task_handle, interpreter):
```

And update all callers to pass in their interpreter.

Reply:
Updated the function signature as suggested. Also passed in `interpreter` from the caller `PersistedTask`.